server/index.js

```diff
@@ -33,41 +33,72 @@ app.use((req, res, next) => {
   next()
 })
 
-// Remove trailing slashes
+// API Docs - resolve before trailing slash redirect so assets don't break
```

Comment:
Unfortunately it's proving extremely frustrating to get the api-docs to work in Express without the trailing slash. We end up with styles broken on either the Express server version, or the `localhost:4567` version.